### DISCUSSION TITLE
Speedify VPN Bypass Not Working?

### DISCUSSION THREAD

**Discussion by taxbeday** [SmoothWAN/SmoothWAN#68] [General & Help]
Does anyone know why the Speedify VPN bypass option for Netflix etc sometimes works and sometimes does not?

**Comment by TalalMash** [comment]
Speedify uses WANs gateway as a DNS for bypass, if the gateway doesn't provide a DNS service and set as primary, bypass won't work, setting the no DNS wan as secondary is a workaround, I've contacted Speedify about this. But what is your WAN configuration?
